docs/scout.txt

```diff
@@ -0,0 +1,198 @@
+.. _laravel-scout:
+
+============================
+Integrated Search with Scout
+============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, code example, text search, atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the Laravel Scout feature in
+your {+odm-long+} application. Scout allows you to implement full-text
+search on your Eloquent models. To learn more, see `Laravel Scout
+<https://laravel.com/docs/{+laravel-docs-version+}/scout>`__ in the
+Laravel documentation.
+
+The Scout Integration for {+odm-long+} provides the following
+functionality:
+
+- Provides an abstraction to create search indexes on documents in
+  MongoDB collections and other search engines. In MongoDB, this feature
```

Comment:
S: I'm a little confused by the grouping of "MongoDB and other search engines", maybe just removing "other" would work
```suggestion
  MongoDB collections and search engines. In MongoDB, this feature
```

Reply:
can use external instead, to match the future section at the end